lib/service/src/main/java/io/atlasmap/service/AtlasLibraryLoader.java

```diff
@@ -89,6 +93,35 @@ public void clearLibaries() {
         reload();
     }
 
+    public ArrayList<String> getClassCandidates() throws AtlasException {
```

Comment:
I would use `getLibraryClassNames()` to align wording with UI - and avoid `candidate` unless there's specific implication, like filtering candidates for some specific purpose.

Reply:
done